docs/spec_draft.md

```diff
@@ -251,20 +251,21 @@ of the following:
   unsigned underflow).
 
 For floating-point element types, it implements the `addition` operation from
-the IEEE-754 specification.
+the IEEE-754 specification. For boolean element types, the behavior is same as
+[stablehlo.xor](#stablehloor).
```

Comment:
From here it seems MHLO and HLO has differences in the semantics of Add.  
 https://github.com/tensorflow/tensorflow/blob/386a9d9cb0dccc146fcfe2d619677598145e336b/tensorflow/compiler/mlir/xla/mlir_hlo_to_hlo.cc#L2195
 
 Looking for reason of the differences.

Reply:
The difference seems to be mainly because of the fact that MLIR (and MHLO for being an MLIR dialect) uses the same integer type `iN` for  booleans and integers and implementing the same XLA or semantics for add needs saturating the addition of i1's on overflow.

StableHLO spec is currently having the XLA behavior.